playground/web/index.html

```diff
@@ -0,0 +1,382 @@
+            font-size: 0.85rem;
+            color: #2c3e50;
+            border-bottom: 1px solid #ddd;
+        }
+
+        .output-content {
+            padding: 0.75rem;
+            background: #fafafa;
+            min-height: 60px;
+            max-height: 200px;
+            overflow-y: auto;
+            font-size: 0.85rem;
+            line-height: 1.4;
+        }
+
+        .match-highlight {
+            background: #e74c3c;
+            color: white;
+            padding: 1px 2px;
+            border-radius: 2px;
+            font-weight: bold;
+        }
+
+        .capture-highlight {
+            background: #9b59b6;
+            color: white;
+            padding: 1px 2px;
+            border-radius: 2px;
+            font-weight: bold;
+        }
+
+        .error {
+            color: #e74c3c;
+            background: #fdf2f2;
+            padding: 0.75rem;
+            border-radius: 4px;
+            border: 1px solid #fecaca;
+            font-size: 0.85rem;
+        }
+
+        .success {
+            color: #27ae60;
+            font-weight: bold;
+        }
+
+        .info {
+            color: #7f8c8d;
+            font-style: italic;
+        }
+
+        .toggle-section {
+            margin-bottom: 1rem;
+        }
+
+        .toggle-btn {
+            background: #3498db;
+            color: white;
+            border: none;
+            padding: 0.5rem 1rem;
+            border-radius: 4px;
+            cursor: pointer;
+            font-size: 0.85rem;
+            transition: background-color 0.2s;
+        }
+
+        .toggle-btn:hover {
+            background: #2980b9;
+        }
+
+        .toggle-btn.active {
+            background: #27ae60;
+        }
+
+        .hidden {
+            display: none;
+        }
+
+        .parse-tree {
+            background: #2c3e50;
+            color: #ecf0f1;
+            padding: 1rem;
+            border-radius: 4px;
+            white-space: pre-wrap;
+            font-family: 'Courier New', monospace;
+            font-size: 0.8rem;
+            line-height: 1.3;
+            overflow-y: auto;
+        }
+
+        .loading {
+            opacity: 0.6;
+            pointer-events: none;
+        }
+
+        @media (max-width: 768px) {
+            .container {
+                grid-template-columns: 1fr;
+            }
+            
+            .flags-container {
+                grid-template-columns: repeat(auto-fit, minmax(120px, 1fr));
+            }
+        }
+
+        footer {
+            text-align: center;
+            padding: 1rem;
+            background-color: #2c3e50;
+            color: #ecf0f1;
+            font-size: 0.9rem;
+            margin-top: 2rem;
+        }
+
+        footer a {
+            color: #ecf0f1;
+            text-decoration: none;
+        }
+
+        #highlighted-text {
+            white-space: pre-line;
+        }
+    </style>
+</head>
+<body>
+    <div class="header">
+        <h1>🎯 fancy-regex Playground</h1>
```

Comment:
I don't like the emoji personally.. Was wondering if we could use something that reminds us of regexes. Maybe `/fancy-regex playground/`?